WFInfo/Data.cs

```diff
@@ -1087,7 +1083,9 @@ public async Task GetUserLogin(string email, string password)
             }
             else
             {
-                throw new Exception("GetUserLogin, " + responseBody + $"Email: {email}, Pw length: {password.Length}");
+                Regex rgx = new Regex("[a-zA-Z0-9]");
+                string censoredEmail = rgx.Replace(email, "*");
+                throw new Exception("GetUserLogin, " + responseBody + $"Email length: {censoredEmail}, Pw length: {password.Length}");
```

Comment:
This is not length tho, this is an email. Perhaps we want to change print `censoredEmail.length()`?

Reply:
Was a previous change, forgot to remove the length again from the debug line